usefulUtilities/webAppTemplate/webRouter..js

```diff
@@ -0,0 +1,113 @@
+
+const express = require('express');
+const router = express.Router();
+const Database = require('./dbClass.js');
+const dbConfig = require("./dbInfo.json").config;
+const mysql = require('mysql');
+
+const connectionConfig = {
+    host: dbConfig.mySQLHost,
+    user: dbConfig.mySQLUsername,
+    password: dbConfig.mySQLPassword,
+    database: dbConfig.databaseName,
+} 
+
+
+router.get("/", (req, res) => {
+    res.send("TEST")
+})
+
+
+router.get('/settings', (req, res) => {
```

Comment:
do you plan to make this path and its response more generic?

Reply:
Fixed to make this file as generic as I could in the other places as well